docs/add-ins/addin-tutorial.md

```diff
@@ -1,107 +1,219 @@
 ---
 
-# Advanced Outlook add-in tutorial: Git the Gist
+# Tutorial: Build a message compose Outlook add-in
 
-The purpose of this guide is to walk through the process of creating an Outlook add-in that allows the user to access their [Gists](https://gist.github.com/) on GitHub. The source code in this [repository](https://github.com/jasonjoh/git-the-gist) is what you should end up with if you follow the steps outlined here.
+This tutorial teaches you how to build an Outlook add-in that can be used in message compose mode to insert content into the body of a message.
 
-This tutorial will use the [Microsoft Office Add-in Project Generator](https://github.com/officedev/generator-office) to generate an add-in project.
+In this tutorial, you will:
+
+> [!div class="checklist"]
+> * Create an Outlook add-in project
+> * Define buttons that will render in the compose message window
+> * Implement a first-run experience that collects information from the user and fetches data from an external service
+> * Implement a UI-less button that invokes a function
+> * Implement a task pane that inserts content into the body of a message
 
 ## Prerequisites
 
-Outlook add-ins are comprised of HTML, CSS, and JavaScript files, so technically the only absolute prerequisite is some sort of web server to host the files. However, in this guide we're going to use some tools to make it easier to get up and running quickly. To follow along with this guide, you'll need the following:
+* [Node.js and npm](https://nodejs.org/en/)
 
-- [Node.js](https://nodejs.org)
-- [Yeoman](https://yeoman.io/)
-- [Microsoft Office Add-in Project Generator](https://github.com/officedev/generator-office)
-- Internet Explorer version 11 or later
+* The latest version of [Yeoman](http://yeoman.io/) and the [Yeoman generator for Office Add-ins](https://www.npmjs.com/package/generator-office). To install these tools globally, run the following command via the command prompt:
 
-> [!TIP]
-> Once you have Node.js installed, you can install all of the other prerequisites via NPM:
->
-> npm install -g yo generator-office
+    ```bash
+    npm install -g yo generator-office
+    ```
 
+* Outlook 2016 or later for Windows (connected to an Office 365 account)
 
-You will also need Outlook 2016 or later connected to an Office 365 account, Outlook.com account, or a Microsoft Exchange Server and a GitHub account to test all of the features of the add-in we're going to create.
+* A [GitHub](https://www.github.com) account 
 
-## Create the add-in
+## Setup
 
-Open your command prompt/shell in an empty directory. Enter `yo office` and follow the prompts.
+The add-in that you'll create in this tutorial will read [gists](https://gist.github.com) from the user's GitHub account and add the selected gist(s) to the body of a message. Complete the following steps to create two new gists that you can use to test the add-in you're going to build.
```

Comment:
suggest replace "gist(s)" with "gists" as screen readers don't usually read this properly